src/admin-views/upsell/default.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Upsell Template
+ * The base template for TEC Upsell notices.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe/upsell/upsell.php
+ *
+ * @since TBD
+ *
+ * @var string        $text      Text of upsell content (excluding link text).
+ * @var string        $link_text Link text.
+ * @var string        $link_url  Link URL.
+ * @var string        $icon_url  URL to icon.
+ * @var array<string> $classes   Additional classes to add to the upsell div.
+ * @var array<string> $link      Array of link properties, including 'text', 'url', 'rel', 'target' and 'classes'.
+ * 
+ * @version TBD
+ */
+
+$upsell_classes = [ 'tec-admin__upsell' ];
+if ( ! empty( $classes ) ) {
+	$upsell_classes = array_merge( $upsell_classes, $classes );
+}
+
+$link_classes = [ 'tec-admin__upsell-link '];
+if ( ! empty( $link ) && ! empty( $link['classes'] ) ) {
+	$link_classes = array_merge( $link_classes, $link['classes'] );
+}
+
+?>
+<div <?php tribe_classes( $upsell_classes ); ?>>
```

Comment:
Probably we'll want to have `tribe-common` as a wrapper (maybe a level up) so that we have the resets, and that we can use the font that we want there.